### PR TITLE
chore: merged mutate and query into one query method

### PR DESCRIPTION
**Pull request by chefsale**
Added a task to fully deleted the mutate and execute calls in the future https://github.com/calimero-network/core/issues/842, tagged them @depracated for now to keep current things working. Also, we should have a breaking SDK and core release to push changes like this. Ideally before 1.0.0.

### CODE REVIEW

File: crates/server-primitives/src/jsonrpc.rs [hunk 1]
```diff
@@ -233,9 +175,8 @@ impl MutateResponse {
...
 #[serde(tag = "type", content = "data")]
 #[non_exhaustive]
-pub enum MutateError {
+pub enum ExecuteError {
     SerdeError { message: String },
     CallError(CallError),
     FunctionCallError(String),
-}
-// *************************************************************************
+}
```
Comment (frdomovic) [inline crates/server-primitives/src/jsonrpc.rs]: eof

**Review by frdomovic — APPROVED** [review]
Looks good, was new sdk version tested with one of the examples?

File: crates/server-primitives/src/jsonrpc.rs (hunk 1 above)
Comment (chefsale) [reply crates/server-primitives/src/jsonrpc.rs]: wdym? :)

Comment (frdomovic) [reply crates/server-primitives/src/jsonrpc.rs]:
```suggestion
}

```
end of file. To match other files in project where all end with space

Comment (chefsale) [reply crates/server-primitives/src/jsonrpc.rs]: Oh 😂

**Pull request merged**